lib/vsc/install/shared_setup.py

```diff
@@ -1438,19 +1438,24 @@ def parse_target(self, target, urltemplate=None):
                     new_target[k] = type(v)()
                     new_target[k] += v
 
+        tests_requires = new_target.setdefault('tests_require', [])
         if sys.version_info < (2, 7):
             # py26 support dropped in 0.8, and the old versions don't detect enough
             log.info('no prospector support in py26 (or older)')
-            tests_requires = new_target.setdefault('tests_require', [])
-            new_target['tests_require'] = [x for x in tests_requires if 'prospector' not in x]
-        else:
+            tests_requires = [x for x in tests_requires if 'prospector' not in x]
+        elif sys.version_info < (3, 0):
             log.info('adding prospector to tests_require')
-            tests_requires = new_target.setdefault('tests_require', [])
+            # limit version to avoid pulling in python 3.x only versions
+            tests_requires.append('zipp < 2.0')
+            tests_requires.append('configparser < 5.0')
```

Comment:
Are these (indirect) test dependencies? Via which pkg are these required?

Reply:
As far as I can tell, pydocstyle is the issue. It pulls in configparser without any restriction.

zipp is unclear. I think it's through pyflakes -> importlib_metadata -> zipp.